actions/setup-gap/envoy.yaml.template

```diff
@@ -61,31 +110,32 @@ static_resources:
                             )
 
                             if response_headers[":status"] == "200" then
-                                request_handle:logInfo("GitHub OIDC token fetched successfully.")
-                                request_handle:logInfo(response_body)
+                                request_handle:logInfo("k8s-api: GitHub OIDC token fetched successfully.")
                                 local data = JSON.decode(response_body)
                                 jwt_token = data.value -- Assign the token value
-                                jwt_expiration = os.time() + 300 -- Tokens are valid for 5 minutes
+                                jwt_expiration = get_jwt_exp(jwt_token)
                             else
-                                request_handle:logErr("Failed to fetch GitHub OIDC token. Status: " .. response_headers[":status"])
+                                request_handle:logErr("k8s-api: Failed to fetch GitHub OIDC token. Status: " .. response_headers[":status"])
                             end
 
                             refreshing = false
                         end
 
                         -- Function to refresh the token if needed
                         function refresh_token_if_needed(request_handle)
-                            local current_time = os.time()
-                            if not jwt_token or jwt_expiration - current_time < 60 then
+                            if not jwt_token or jwt_expiration - os.time() < 60 then
```

Comment:
@chainchad I'm unsure about refreshing the token if it is about to expire in less than 60 seconds. Maybe it's too frequent? Thoughts?